tests/components/google_assistant/test_google_assistant.py

```diff
@@ -381,3 +461,12 @@ async def test_execute_request(hass_fixture, assistant_client, auth_header):
 
     lounge = hass_fixture.states.get("media_player.lounge_room")
     assert lounge.state == "off"
+
+    humidifier_state = hass_fixture.states.get("humidifier.humidifier")
+    assert humidifier_state.state == "off"
+
+    dehumidifier = hass_fixture.states.get("humidifier.dehumidifier")
+    assert dehumidifier.attributes.get(humidifier.ATTR_HUMIDITY) == 45
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)